Source/ASTableNode.mm

```diff
@@ -892,3 +892,39 @@ - (void)waitUntilAllUpdatesAreCommitted
 }
 
 @end
+
+#pragma mark - UIGestureRecognizerDelegate
+
+@implementation ASTableNode (UIGestureRecognizerDelegate)
+
+- (BOOL)gestureRecognizerShouldBegin:(UIGestureRecognizer *)gestureRecognizer
+{
+  return [super gestureRecognizerShouldBegin:gestureRecognizer];
```

Comment:
Looks like this is different than the behavior in [ASCollectionNode](https://github.com/TextureGroup/Texture/pull/1457/files#diff-d505487f1d5096efe0b8a6cd4ae539ceR1047). Should we change the behavior on ASCollectionNode or here?

Reply:
Might be worth copying over the comment in ASCollectionNode too?

Reply:
As the subclass of `ASDisplayNode`, it should be supervised by the implementation of `-[gestureRecognizerShouldBegin:]` from its superclass. Is that applicable?